Standards/scs-0210-v2-k8s-version-policy.md

```diff
@@ -0,0 +1,75 @@
+---
+title: SCS K8S Version Policy
+type: Standard
+status: Draft
+stabilized_at: 2023-08-16
+track: KaaS
+---
+
+## Introduction
+
+The Kubernetes project maintains multiple release versions including their patched versions.
+In the project, the three most recent minor releases are actively maintained, with a fourth
+version being in development. As soon as a new minor version is officially released,
+the oldest version is dropped out of the support period.
+Kubernetes supports its releases for around 14 months. 12 of these are the standard
+support period. The remaining 2 months are used for:
+
+- CVEs (under the advisement of the Security Response Committee)
+- dependency issues (including base image updates)
+- critical core component issues
+
+[Kubernetes Support Period](https://kubernetes.io/releases/patch-releases/#support-period)
+
+The Kubernetes releases are planned for about 4 months, which usually results in about
+**3 minor** releases per year.
+[Kubernetes Release Cycle](https://kubernetes.io/releases/release/#the-release-cycle)
+
+Patches to these releases are provided monthly, with the exception of the first patch,
+which is usually provided 1-2 weeks after the initial release.
+[Patch Release Cadence](https://kubernetes.io/releases/patch-releases/#cadence)
+
+## Motivation
+
+Kubernetes is a living, fast-paced project, which follows a pre-defined release cycle.
+This enables forward planning with regards to releases and patches, but also implies a
+necessity to upgrade to newer versions quickly.
+
+We want to achieve an up to date policy, which means that providers should be mostly in sync with the upstream and don't
+fall behind the official Kubernetes releases. In our opinion, this should be achievable, since new versions are released
+periodical on a well communicated schedule. This should enable processes to be set up around this.
+Being up to date ensures, that security issues and bugs are addressed and new features are made available when using SCS compliant clusters.
+
+On the other hand, providers should support minor Kubernetes versions at least as long as
+the official support period is, since users could depend on specific versions.
```

Comment:
OTOH, since users may depend on specific versions, providers should support minor Kubernetes versions *for the duration of the official support period*.

Reply:
Agree. I wrote this into the document in the latest commit.